src/main/java/io/cdap/plugin/batch/sink/MongoDBBatchSink.java

```diff
@@ -115,9 +115,9 @@ public void transform(StructuredRecord record, Emitter<KeyValue<NullWritable, BS
     emitter.emit(new KeyValue<>(NullWritable.get(), bsonWritable));
   }
 
-  private void validateInputSchema(Schema inputSchema) {
+  private void validateInputSchema(MongoDBSinkConfig config, Schema inputSchema) {
```

Comment:
this isn't a static method, so no need to pass in the config here

Reply:
Done.